lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -14,8 +14,24 @@ module Resources
         'status': 'generated',
         'div': '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Related Person</b></p><p><b>Name</b>: SMART, HAILEY'\
                '</p><p><b>DOB</b>: Dec  2, 2003</p><p><b>Gender</b>: female</p><p><b>Patient</b>: SMART, NANCY</p>'\
-               '<p><b>Status</b>: Active</p><p><b>Relationship</b>: Family Member</p></div>'
+               '<p><b>Status</b>: Active</p><p><b>Relationship</b>: Family Member</p>'\
+               '<p><b>Relationship Level</b>: Patient Level Relationship</p></div>'
       },
+      'extension': [
+        {
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/relationship-level',
+          'valueCodeableConcept': {
+            'coding': [
+              {
+                'system': 'http://hl7.org/fhir/resource-types',
+                'code': 'Patient',
+                'display': 'Patient Level Relationship'
```

Comment:
This should be the display from the system, thus needs to change to Patient

Reply:
Fixed: https://github.com/cerner/fhir.cerner.com/pull/480/commits/98af69a03800c29aa7b196f80be2dd2bd7ab0ee1#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR28
Also fixed it in the narratives as well